content/applications/hr/attendances/hardware.rst

```diff
@@ -2,56 +2,114 @@
 Hardware
 ========
 
-Kiosk management
-================
+Employees who are **not** database users, and therefore do not have access to the **Attendances**
+app, must sig in and out of work using a kiosk. The following are the physical requirements for
+setting up a kiosk.
 
-A kiosk is a self-service station that allows employees to check in and check out for work shifts.
+Kiosk devices
+=============
 
-There are two ways to set up a kiosk:
+A kiosk is a self-service station where employees can check in and out of work with either a
+:ref:`badge <attendances/hardware/badges>` or an :ref:`RFID key fob <attendances/hardware/rfid>`.
+Typically, these devices are dedicated as kiosks only, but any device with an internet browser is
+able to be setup as a kiosk.
 
-- **Laptop and desktop PC**
+A kiosk is used by navigating to the webpage specified in the :ref:`configuration
+<attendances/kiosk-settings>` section of the *Attendances* app.
 
-  Running a kiosk in a web browser is the cheapest and most flexible option. You can print employee
-  badges with any thermal or inkjet printer compatible with your web browser.
+Kiosks are setup using one of the following types of devices:
 
-- **Tablet and mobile phone (Android or iOS)**
+- Laptop PC
+- Desktop PC
+- Tablet
+- Mobile phone (Android or iOS)
 
-  Tablets and mobile phones take up much less space, and their touchscreens are easy to use.
-  Consider putting them in a secure stand at the front desk or mounting them securely on a wall.
+.. tip::
+   Touchscreens are easy to us, and tablets and mobile phones take up less space. Consider using a
+   smaller device with a touchscreen as a kiosk.
 
-  .. tip::
-     We recommend using an iPad together with the `Heckler Design WindFall Stand
-     <https://hecklerdesign.com/products/windfall-stand-for-ipad>`_
+   It is recommended to place kiosks in a secure stand where employees check in and out, or
+   mount them securely on a wall.
 
-RFID key fob readers
-====================
+   One recommended setup is using an iPad together with the `Heckler Design WindFall Stand
+   <https://hecklerdesign.com/products/windfall-stand-for-ipad>`_.
```

Comment:
Who recommends this specific product? I would consider removing this.

Reply:
This was in the original documentation- both this and the scanner. I didn't think to remove these since they were original, but they might be outdated. I do agree- it is odd to include a product, so I will remove these .